### DISCUSSION TITLE
How can I display hls stream on the web in ubuntu?

### DISCUSSION THREAD

**Discussion by o3omoomin** [bluenviron/mediamtx#3082] [Questions]
### Question

Using Mediamtx, I worked on rtsp to hls on Mac.
I created an hls video stream server called http://localhost:8080/camera2/ and

I tried to play this in the Ubuntu environment, but the codec error below was displayed.

```
stream makes use of codecs which are incompatible with this browser or operative system

```

I want to show the above hls video stream in react in ubuntu environment.
Below is the code I wrote

```
<iframe src="http://192.168.0.37:8888/camera2/" scrolling="no"></iframe>

```

**Comment by aler9** [comment]
Hello, the message "stream makes use of codecs which are incompatible with this browser or operative system" means exactly "stream makes use of codecs which are incompatible with this browser or operative system". Change the codecs of your stream in order to use some codecs that are natively compatible with Ubuntu and the browser you are using, that are probably H264 or AV1 (but not H265).

[a comment by an automated account is not shown]
